### DISCUSSION TITLE
Cased extrusions

### DISCUSSION THREAD

**Discussion by kircher1** [maplibre/maplibre-gl-js#1653] [Ideas]
Is a cased extrusion possible or something that has previously been investigated?

| Current fill extrusion | What a cased extrusion could look like| 
| --- | --- |
|  ![image](https://user-images.githubusercontent.com/20366429/190723328-63dffcb5-0ec3-468f-9312-16c0b063f63b.png) |  ![image](https://user-images.githubusercontent.com/20366429/190723368-bb91e629-cc4a-413f-8a16-e1fa24b2a6ba.png) |

Besides being a potentially useful styling feature, it could also provide an avenue for anti-aliasing the extrusion edges (i.e. using the line for AA) rather than resorting to MSAA.

**Comment by HarelM** [comment]
Worth asking is slack.
I can't remember seen something similar...
